src/app/frontend/deploy/helpsection/userhelp.scss

```diff
@@ -0,0 +1,32 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+@import "../../../assets/styles/color_variables";
+
+.kd-user-help * {
+  color: gray;
```

Comment:
Try to always extract colors to variables. If this is something that can be used in  more places then it can be places in external file, if not then just declare variables in file.